Sources/Testing/Running/XCTestScaffold.swift

```diff
@@ -168,6 +195,20 @@ public enum XCTestScaffold: Sendable {
 #endif
     }
 
+    // If the SWT_SELECTED_TEST_IDS environment variable is set, split it into
+    // test IDs (separated by ";", test ID components separated by "/") and set
+    // the configuration's test filter to match it.
+    //
+    // This environment variable stands in for `swift test --filter`.
+    let testIDs: [Test.ID]? = Environment.variable(named: "SWT_SELECTED_TEST_IDS").map { testIDs in
+      testIDs.split(separator: ";").map { testID in
+        Test.ID(testID.split(separator: "/").map(String.init))
```

Comment:
Might want to protect against empty strings in a component after splitting?

Reply:
Ah, sure.